submissions/DawoduRahmon/HTML/product-page.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+<html lang="chinese">
```

Comment:
When you are specifying the language attribute, the value you should use should be the language's ISO code. E.g. English is `en` while France is `fr`. So use the proper ISO code.